internal/commands/build.go

```diff
@@ -151,6 +162,7 @@ If not set (or set to empty string) the standard socket location will be used.
 Special value 'inherit' may be used in which case DOCKER_HOST environment variable will be used.
 This option may set DOCKER_HOST environment variable for the build container if needed.
 `)
+	cmd.Flags().StringVar(&buildFlags.LifecycleImage, "lifecycle-image", cfg.LifecycleImage, `Custom lifecycle image to use. If not provided a lifecycle image from "buildpacksio/lifecycle" will be used.`)
```

Comment:
```suggestion
	cmd.Flags().StringVar(&buildFlags.LifecycleImage, "lifecycle-image", cfg.LifecycleImage, `Custom lifecycle image to use for analysis, restore, and export when builder is untrusted. If not provided a lifecycle image from "buildpacksio/lifecycle" will be used.`)
```

Reply:
IDK if this is too verbose... I was just trying to get the point across that it's not used ALL the time.

Reply:
I used "The image to use when executing sensitive phases." on the [Tekton task](https://github.com/buildpacks/tekton-integration/tree/main/task/buildpacks-phases/0.2#parameters). Not sure it's much better. 🤦‍♂️